.travis.yml

```diff
@@ -38,12 +40,38 @@ before_install:
 
 - if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
         mkdir -p $HOME/gcloud &&
-        wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud &&
+        wget https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud &&
```

Comment:
The build kept printing a warning about having an outdated version of gcloud, and when it was on the `release/` it prevented me from running `gcloud components update` or getting the `app` component installed.